### ISSUE TITLE
[Payment Due Mon 19 April] Expensify.cash WCAG 2.0 Accessibility Audit/Testing

### ISSUE THREAD

**Issue by rashmicodebase** [Expensify/Expensify.cash#2400]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/master/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Expected Result:
Expensify.cash product should be WCAG 2.0 compliant.

## Actual Result:
Expensify.cash is not meeting all WCAG 2.0 guidelines.

## Action Performed:
Expensify.cash WCAG 2.0 Accessibility Audit/Testing

## Workaround:
No workaround, there are many blockers and critical issues that would impact the disabled users in accessing the platform features.

## Platform:
Where is this issue occurring?

 Web
 iOS
 Android
 Desktop App(MAC)
 Mobile Web

**Version Number:**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
The attached document contains all the necessary information.
[Expensify.csh audit.xlsx](https://github.com/Expensify/Expensify.cash/files/6313793/Expensify.csh.audit.xlsx)

**Expensify/Expensify Issue URL:**

**Comment by laurenreidexpensify** [comment]
@kadiealexander Have you already issued payment for this, I couldn't work it out in Upwork? If so, can we close? Thanks!

**Comment by kadiealexander** [comment]
All payments are completed automatically, so I've closed the contract. Thanks @rashmicodebase!
